pandas/core/generic.py

```diff
@@ -369,14 +369,6 @@ def _constructor(self: FrameOrSeries) -> Type[FrameOrSeries]:
         """
         raise AbstractMethodError(self)
 
-    @property
-    def _constructor_sliced(self):
-        """
-        Used when a manipulation result has one lower dimension(s) as the
-        original, such as DataFrame single columns slicing.
-        """
-        raise AbstractMethodError(self)
-
```

Comment:
This isn't used on Series. Calling it on a Series should just result in an AttributeError.